library/talipot-python/plugins/import/GraphMLImport.py

```diff
@@ -71,6 +71,9 @@ def startElement(self, name, attrs):
                     self.attrLabel = 'label'
 
                 # Create a Talipot property compatible with the attribute type
+                while self.graph.existProperty(attrName):
+                    attrName = '_'+attrName
```

Comment:
Please add spaces around the `+` operator.

You should try the [`flake8`](http://flake8.pycqa.org/en/latest/) tool by the way, it really helps
to write clean Python code and it is really easy to integrate it in IDE.